README.md

```diff
@@ -66,4 +70,21 @@ The code includes a phase classification network with:
 * LSTM temporal model.
 * Pytorch dataset/loader based on the protobufs defined in the SAGES 20' video/data annotation workshop in Houston.
 
-The main training script is under [src/scripts/train_temporal_model.py](scripts/train_temporal_model.py).
+The main training script is under [src/phase_net/train_baseline.py](src/phase_net/train_baseline.py).
```

Comment:
baseline is very generic, baseline for what? Are you saving the original for safety? If so, rename to "old_temporal" vs "temporal", etc